tools/eslint-plugin/dist/configs/imodeljs-recommended.js

```diff
@@ -251,7 +251,7 @@ module.exports = {
     "no-eval": "error",
     "no-fallthrough": "error",
     "no-invalid-this": "off",
-    "no-multiple-empty-lines": "error",
+    "no-multiple-empty-lines": ["error", { max: 1 }],

```

Comment:
@calebmshafer here's the rule. It previously allowed 2 blank lines. I made it allow only 1 blank line like our TSLint rule